ipc/nodepart/arwenDriver.go

```diff
@@ -348,15 +381,25 @@ func (driver *ArwenDriver) continuouslyCopyArwenLogs(arwenStdout io.Reader, arwe
 			}
 
 			line = strings.TrimSpace(line)
-			driver.nodeLogger.Error("ARWEN-ERR", "line", line)
+			driver.arwenMainLogger.Error("ARWEN-ERR", "line", line)
+		}
+	}()
+
+	go func() {
+		for {
+			err := logger.ReceiveLogThroughPipe(driver.arwenMainLogger, arwenLog, driver.logsMarshalizer)
```

Comment:
duplicated code - move to a separate func

Reply:
Fixed.